build-monitor-acceptance/src/main/java/com/smartcodeltd/aether/ArtifactTransporter.java

```diff
@@ -27,9 +30,12 @@
 
 public class ArtifactTransporter {
 
+    private static final Logger Log = LoggerFactory.getLogger(FindFreePort.class);
```

Comment:
I agree that using a Logger is a better idea, but what's the rationale behind registering it as a FindFreePort.class Logger instead of ArtifactTransporter.class one?